research/validation/m1b_export_llamagen_decoder_onnx.py

```
@@ -0,0 +1,145 @@
+    try:
+        torch.onnx.export(
+            wrapper,
+            (tokens,),
+            str(out_path),
+            input_names=["tokens"],
+            output_names=["image"],
+            dynamic_axes=None,
+            opset_version=args.opset,
+            do_constant_folding=True,
+        )
+    except Exception as error:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)